README.md

```diff
@@ -9,7 +9,7 @@ MIT
 [![Platform](https://img.shields.io/cocoapods/p/SwiftDDP.svg?style=flat)](http://cocoapods.org/pods/SwiftDDP)
 
 ## Changelog
-### 0.2.2: 
+### 0.2.2:
```

Comment:
Sorry for the noise in this file, my text editor ate all the trailing whitespace :)